File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/HttpPerformanceData.cs
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class HttpPerformanceData
{
	[JsonProperty("start_timestamp")]
	public double StartTimestamp { get; set; }

	[JsonProperty("duration")]
	public double Duration { get; set; }

	[JsonProperty("duration_to_request_sent")]
	public double? DurationToRequestSent { get; set; }

	[JsonProperty("duration_to_first_byte")]
	public double? DurationToFirstByte { get; set; }

	[JsonProperty("bytes_sent")]
	public long BytesSent { get; set; }

	[JsonProperty("bytes_rcvd")]
	public long BytesReceived { get; set; }
}

File: Android/Vine-5.50.1/Decompilation/src/java/co/vine/android/scribe/model/HTTPPerformanceData.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
/* loaded from: classes.dex */
public class HTTPPerformanceData {

    @JsonField(name = {"bytes_rcvd"})
    public Long bytesReceived;

    @JsonField(name = {"bytes_sent"})
    public Long bytesSent;

    @JsonField(name = {"duration"})
    public Double duration;

    @JsonField(name = {"duration_to_first_byte"})
    public Double durationToFirstByte;

    @JsonField(name = {"duration_to_request_sent"})
    public Double durationToRequestSent;

    @JsonField(name = {"start_timestamp"})
    public Double startTimestamp;
}
